src/InputText/index.jsx

```diff
@@ -8,7 +8,7 @@ function Text(props) {
     <input
       id={props.id}
       className={classNames(props.className)}
-      type="text"
+      type={props.type || 'text'}
```

Comment:
While this does the job of `defaultProps`, I think it might be useful to add a `defaultProps` for the `type` prop.

I think it's useful because we only would keep the `props.type || 'text'` logic to deal with case where `type` *is* defined (but `null`) not where `type` is defined or not.

Basically, while there would be no difference in the behavior of the component, there is a subtle difference in the intention of `defaultProps` vs. `props.type || 'text'` (in my opinion at least)

Reply:
I see where you're coming from on this one, but it's not within the scope at the moment for this task. Willing to revisit once we get this a11y page out the door.